src/parallel/LoadBalancer.h

```diff
@@ -8,8 +8,20 @@
 #include <array>
 #include <tuple>
 
+/**
+ * LoadBalancer class for the usage of arbitrary load balancing classes that are handled by GeneralDomainDecomposition.
+ */
 class LoadBalancer {
 public:
+	/**
+	 * Virtual destructor.
+	 */
 	virtual ~LoadBalancer() = default;
+
+	/**
+	 * The rebalancing call.
```

Comment:
This "documentation" documents nothing...

Reply:
doc